engine/src/test/scala/cromwell/webservice/routes/wes/WesRouteSupportSpec.scala

```diff
@@ -149,4 +154,53 @@ class WesRouteSupportSpec extends AsyncFlatSpec with ScalatestRouteTest with Mat
         rejection shouldEqual MethodRejection(POST)
       }
   }
-}
+
+  behavior of "WES API /runs POST endpoint"
+  it should "return 201 for a successful workflow submission" in {
+    val workflowSource = Multipart.FormData.BodyPart("workflow_url", HttpEntity(MediaTypes.`application/json`, "https://raw.githubusercontent.com/broadinstitute/cromwell/develop/womtool/src/test/resources/validate/wdl_draft3/valid/callable_imports/my_workflow.wdl"))
+    val workflowInputs = Multipart.FormData.BodyPart("workflow_params", HttpEntity(MediaTypes.`application/json`, HelloWorld.rawInputs.toJson.toString()))
+    val formData = Multipart.FormData(workflowSource, workflowInputs).toEntity()
+    Post(s"/ga4gh/wes/$version/runs", formData) ~>
+      wesRoutes ~>
+      check {
+        assertResult(
+          s"""{
+             |  "id": "${CromwellApiServiceSpec.ExistingWorkflowId.toString}",
+             |  "status": "Submitted"
+             |}""".stripMargin) {
+          responseAs[String].parseJson.prettyPrint
+        }
+        assertResult(StatusCodes.Created) {
+          status
+        }
+        headers should be(Seq.empty)
+      }
+  }
+
+
+  behavior of "WES API /runs GET endpoint"
+  it should "return results for a good query" in {
+    Get(s"/ga4gh/wes/v1/runs?status=Succeeded&id=${CromwellApiServiceSpec.ExistingWorkflowId}") ~>
```

Comment:
Can you remind me which API this is for? I don't see GET `/runs` endpoint accepting `status` and `workflowId` parameters

Reply:
These are for the Wes Api for the GET /runs endpoint. The endpoint returns an array that contains elements with "run_id" and "state", so my idea here was to check that we are getting back those two parameters from the query.

Reply:
Right. But I don't think GET `/runs` accepts `status` and `id` as query parameters no? Here the request should be this?
```
Get("/ga4gh/wes/v1/runs") ~>
```

Reply:
I see. Refactoring now...